resources/home/dnanexus/run_workflows/tests/test_manage_dict.py

```
@@ -598,88 +599,94 @@
-        "all_analysis_output": "INPUT-parent_out_dir",
-        "custom_coverage": True,
-        "eggd_multiqc_config_file": {
-            "$dnanexus_link": "file-G0K191j433Gv6JG63b43z8Gy"
-        },
-        "sample_name_prefix": "INPUT-SAMPLE-PREFIX",
-        "sample_name": "INPUT-SAMPLE-NAME",
-        "output_path": "INPUT-analysis_1-out_dir",
-        "run_sample_sheet": {"$dnanexus_link": "INPUT-SAMPLESHEET"},
-    }
+    @pytest.fixture
+    def other_inputs(self, monkeypatch):
```

Ruff (0.8.2)

Missing return type annotation for public function `other_inputs`

(ANN201)